README.md

```diff
@@ -91,7 +91,7 @@ For information on how to install for use with the cloud environment, please see
 
 1. Fork our repo from https://github.com/publiclab/plots2.
 2. In the console, download a copy of your forked repo with `git clone https://github.com/your_username/plots2.git` where `your_username` is your GitHub username.
-3. Enter the new **plots2** directory with `cd plots2`.
+3. Enter the new **plots2** directory with `cd plots2` and then enter `git remote add upstream https://github.com/publiclab/plots2.git` in the command line.
```

Comment:
3. Enter the new **plots2** directory with `cd plots2`.
4. Set your upstream remote: `git remote add upstream https://github.com/publiclab/plots2.git`.